### ISSUE TITLE
Log Archive Full Percent in MAP Elites

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#666]
Make a separate plot command that tracks the percentage of the archive that is filled. This should not require any additional logging, just an extra command in the plot file. First, you have to ask for the total number of bin labels. Let's say, for example, that this number were 10000. Let's also say that in the log file, the number of filled bins is tracked in column 2 (is this right?). Then the quantity you would plot in gnuplot is `1:($2 / 10000)`

In other words, you need to directly print the total archive size directly into the plot file and divide by it

**Comment by MBatt1** [comment]
![image](https://user-images.githubusercontent.com/53229958/120389595-3ca12500-c2f2-11eb-93a6-45ddd6447770.png)
Working, shows decimal percentage instead of actual bin quantity
